index.js

```diff
@@ -0,0 +1,48 @@
+const Discord = require("discord.js");
+const client = new Discord.Client();
+const config = require("./config.json");
+// Messages
+const help = require("./messages/help.json");
+const welcome = require("./messages/welcome.json");
+
+client.on("ready", () => {
+  console.log("I am ready!");
+});
+
+client.on("message", message => {
+  if (message.author.bot) return;
+  if (message.content.indexOf(config.prefix) !== 0) return;
+  if (!message.member.roles.some(r => ["Administrators"].includes(r.name)))
+    return message.reply("Sorry, you don't have permissions to use this!");
+
+  const args = message.content
+    .slice(config.prefix.length)
+    .trim()
+    .split(/ +/g);
+  const command = args.shift().toLowerCase();
+
+  if (command === "help") {
+    let [person, cmd] = args;
```

Comment:
`cmd` or `command`?

Reply:
Command is the actual command used.
`!fed help @zachshallbetter channel`

`!fed` is the prefix of the command that bot picks up.
`help` is the command.
`@zach...` is the user we are targeting.
`channel` is cmd or the actual message we are sending to the user. I can rename it to `msg` to avoid the confusion.

Reply:
We should throw that into the documentation.

Reply:
Do we want to use comments for documentation or are we going to use the wiki, or something else ?